locations/spiders/wefix_fr_be.py

```
@@ -0,0 +1,24 @@
+class WeFixFRBESpider(SitemapSpider, StructuredDataSpider):
+    name = "wefix_fr_be"
+    item_attributes = {"brand": "WeFix", "brand_wikidata": "Q121543865"}
+    sitemap_urls = ["https://boutique.wefix.net/locationsitemap1.xml"]
+    sitemap_rules = [(r"https\:\/\/boutique\.wefix\.net\/.+$", "parse_sd")]
+    wanted_types = ["LocalBusiness"]
+    drop_attributes = {"image", "twitter", "facebook"}
+
+    def post_process_item(self, item: Feature, response: TextResponse, ld_data: dict, **kwargs) -> Iterable[Feature]:
+        apply_category(Categories.SHOP_MOBILE_PHONE, item)
+        if item.get("phone") == "+33 1 76 50 76 50":
+        	item.pop("phone", None)
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)